### DISCUSSION TITLE
Copy type to clipboard

### DISCUSSION THREAD

**Discussion by rdavison** [ocamllabs/vscode-ocaml-platform#1172] [General]
Would it be possible to add support for copying the type of a value to the clipboard? I remember this was possible with merlin on vim: https://github.com/ocaml/merlin/blob/master/vim/merlin/doc/merlin.txt#L76

Perhaps [this vs code plugin](https://github.com/youngvform/copy-hover-type) could be used for some inspiration on how to do this?

Currently, the only way I know how to do this is to physically move the mouse to the hover window and then clicking and dragging to the spot where the documentation begins.

Alternatively, would it be possible to autocomplete function arguments? It would be nice to have some way of "destructing" a function and having the cursor jump to each of the arguments automatically.

**Comment by lukstafi** [comment]
Thanks a lot for pointing me to [Copy Hover Type](https://github.com/youngvform/copy-hover-type)! Is it insufficient?
